oras.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: oras
+  version: 0.16.0
+  epoch: 0
+  description: OCI registry client - managing content like artifacts, images, packages.
+  target-architecture:
+    - all
+  copyright:
+    - license: Apache-2.0
+      paths:
+        - "*"
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/oras-project/oras
+      branch: v${{package.version}}
+      destination: oras

```

Comment:
Can we use `fetch` here, or `expected-commit` that [someone very smartly added recently](https://github.com/chainguard-dev/melange/pull/275) 😄

Reply:
oras uses git commit in ldflags in their Makefile. that's why i use git-checkout instead of fetch.